src/app_engine/client.py

```diff
@@ -2,25 +2,35 @@
 #
 # Copyright 2015 Google Inc. All Rights Reserved.
 
-"""Client model
+"""Client model.
 
 This module implements the client data model.
 """
 
+import uuid
+
+
 # For now we have (room_id, client_id) pairs are 'unique' but client_ids are
```

Comment:
Please update the comment and explain the difference between client_id and session_id.